code/__DEFINES/preferences.dm

```diff
@@ -131,6 +131,9 @@
 /// such as hair color being affixed to hair.
 #define PREFERENCE_CATEGORY_SUPPLEMENTAL_FEATURES "supplemental_features"
 
+/// These preferences will not be rendered on the preferences page, and are practically invisible unless specifically rendered.
+#define PREFERENCE_CATEGORY_NON_APPLICABLE "non_applicable"
```

Comment:
I don't think this makes sense and think they should instead be their own category

Reply:
rolled into PREFERENCE_CATEGORY_MANUALLY_RENDERED, but if you want i can specify it into PREFERENCE_CATEGORY_QUIRK_FEATURES - i just put it in manually rendered since someone in the future may want to manually render something and thus itd be helpful to have a category for that specifically